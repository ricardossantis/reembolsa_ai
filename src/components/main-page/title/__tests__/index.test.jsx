import React from 'react';
import Title from '../index';
import renderer from 'react-test-renderer';

it("renderer", () =>{
    const tree = renderer
    .create(<Title bckTitle="#365083" colorTitle="#FFFFFF">{'Test'}</Title>)
    .toJSON();

    expect(tree).toMatchSnapshot();
});