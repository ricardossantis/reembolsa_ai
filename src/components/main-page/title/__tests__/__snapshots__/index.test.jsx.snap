// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderer 1`] = `
<div
  className="sc-bdnylx ckyYMA"
  style={
    Object {
      "backgroundColor": "#365083",
      "color": "#FFFFFF",
    }
  }
>
  Test
</div>
`;
